app/services/service.rb

```diff
@@ -0,0 +1,6 @@
+class Service
+  def self.get_events(zip_code)
+    response = Faraday.get "http://api.eventful.com/json/events/search?app_key=8gWGVrJnPdvtqFrq&location=#{zip_code}&sort_order=popularity&page_size=16"
```

Comment:
this is the correct endpoint with the correct parameters 👍 

i'd suggest maybe setting a few of the parameters as variables above so that they are easier to change. you could also use the `Faraday.new` syntax and set it as a variable, then add the parameters you want to send in after you've defined this. this is much easier to read!

you could do something like this:

``` 
  def self.get_events(zip_code)
    conn = Faraday.new(url: 'http://api.eventful.com/json/events/search') do |faraday|
      faraday.adapter  Faraday.default_adapter
    end
    response = conn.get do |req|
      req.params['app_key']    = "8gWGVrJnPdvtqFrq"
      req.params['location']   = zip_code
      req.params['date']       = 'March'
      req.params['sort_order'] = 'popularity'
      req.params['page_size'] =   16
    end
    parsed = JSON.parse(response.body, symbolize_names: true)[:events][:event]
  end